packages/electron-chrome-extensions/src/browser/popup.ts

```diff
@@ -176,14 +179,26 @@ export class PopupView {
   }
 
   private updatePosition() {
-    if (!this.browserWindow || !this.parent) return
+    if (!this.browserWindow || !this.sender) return
+
+    if (this.sender) {
+      const webcontents = webContents.fromId(this.sender.id)
+
+      if (webcontents) {
+        const window = BrowserWindow.fromWebContents(webcontents)
+        if (window) {
+          const winBounds = window.getBounds()
+          console.log(winBounds);
+          this.anchorRect.x = this.anchorRect.x + winBounds.x
+          this.anchorRect.y = this.anchorRect.y + winBounds.y
```

Comment:
if updatePosition is called multiple times, won't this result in anchorRect accumulating over time?

Rather than changing what anchorRect is, it may be better to instead do this calculation in a local variable and not change what anchorRect represents.

Reply:
this.anchorRect will have the value relative to the sender's window. So hence added in it. If we declare a local variable then that we will not be able to fetch for calculation of x as its const variable. We need to go let or var in that case.